src/main/java/lotto/domain/Common.java

```diff
@@ -0,0 +1,8 @@
+package lotto.domain;
+
+public final class Common {
+	public static final int LOTTO_PRICE = 1000;
```

Comment:
습관적으로 언더바(_)를 사용해서 숫자의 단위 가독성에 신경써보면 좋을 것 같아여~
```suggestion
	public static final int LOTTO_PRICE = 1_000;
```

Reply:
네, 한눈에 쉽게 들어와서 그냥 넘어갔네요. 앞으로는 일관적으로 언더바를 넣는걸로 할게요 👍